site/docs/pages/wallet/wallet-dropdown-fund-link.mdx

```diff
@@ -0,0 +1,191 @@
+  Identity,
+  EthBalance,
+} from '@coinbase/onchainkit/identity';
+import { color } from '@coinbase/onchainkit/theme';
+
+export function WalletComponents() {
+  return (
+    <div className="flex justify-end">
+      <Wallet>
+        <ConnectWallet>
+          <Avatar className="h-6 w-6" />
+          <Name />
+        </ConnectWallet>
+        // [!code focus]
+        <WalletDropdown>
+          // [!code focus]
+          <Identity className="px-4 pt-3 pb-2" hasCopyAddressOnClick>
+            <Avatar />
+            <Name />
+            <Address className={color.foregroundMuted} />
+            <EthBalance />
+          </Identity>
+          <WalletDropdownFundLink /> // [!code focus]
+          <WalletDropdownLink icon="wallet" href="https://wallet.coinbase.com">
+            Wallet
+          </WalletDropdownLink>
+          <WalletDropdownDisconnect />
+        </WalletDropdown>
+      </Wallet>
+      // [!code focus]
+    </div>
+  );
+}
+```
+
+<WalletComponents>
+  <Wallet>
+    <ConnectWallet>
+      <Avatar className="h-6 w-6" />
+      <Name />
+    </ConnectWallet>
+    <WalletDropdown>
+      <WalletDropdownFundLink />
+      <WalletDropdownLink icon="wallet" href="https://wallet.coinbase.com">
+        Wallet
+      </WalletDropdownLink>
+      <WalletDropdownDisconnect />
+    </WalletDropdown>
+  </Wallet>
+</WalletComponents>
+
+### Override text
+
+You can override component text using the `text` prop.
+
+```tsx
+// omitted for brevity
+
+<Wallet>
+  <ConnectWallet>
+    ...
+  </ConnectWallet>
+  <WalletDropdown>
+    ...
+    <WalletDropdownFundLink text="Add crypto" /> // [!code focus]
+  </WalletDropdown>
+</Wallet>
+```
+
+<WalletComponents>
+  <Wallet>
+    <ConnectWallet>
+      <Avatar className="h-6 w-6" />
+      <Name />
+    </ConnectWallet>
+    <WalletDropdown>
+      <WalletDropdownFundLink text="Add crypto" />
+      <WalletDropdownDisconnect />
+    </WalletDropdown>
+  </Wallet>
+</WalletComponents>
+
+### Open in window
+
+You can override default link behavior to open the funding flow in a popup window by using the `type` prop.
+
+```tsx
+// omitted for brevity
+
+<Wallet>
+  <ConnectWallet>
+    ...
+  </ConnectWallet>
+  <WalletDropdown>
+    ...
+    <WalletDropdownFundLink type="window" /> // [!code focus]
+  </WalletDropdown>
+</Wallet>
+```
+
+<WalletComponents>
+  <Wallet>
+    <ConnectWallet>
+      <Avatar className="h-6 w-6" />
+      <Name />
+    </ConnectWallet>
+    <WalletDropdown>
+      <WalletDropdownFundLink type="window" />
+      <WalletDropdownDisconnect />
+    </WalletDropdown>
+  </Wallet>
+</WalletComponents>
+
+### Customizing the window size
+
+When using popup mode, you're able to customize the size of the popup window using the `size` prop. Valid values are `s`, `m`, and `l`.
+
+```tsx
+// omitted for brevity
+
+<Wallet>
+  <ConnectWallet>
+    ...
+  </ConnectWallet>
+  <WalletDropdown>
+    ...
+    <WalletDropdownFundLink type="window" size="s" /> // [!code focus]
```

Comment:
We should not use `size` as prop, because this is a special word we want to use across all OnchainKit.

When you build a Design System you want to think 7 steps ahead. This implementation things only 3 steps ahead, and there is the risk of losing the game and lose our queen.

What's a better API for this?

Reply:
`windowSize`?

Reply:
I'm not sure what the plans are for `size` across OnchainKit but I imagine it'd be similar to this use case

Reply:
Maybe this can be the first implementation!

Reply:
changed to `windowSize` for now